projects/cli-files/go-ls/cmd/root.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+)
+
+func Execute() {
+	arg := "."
+	if len(os.Args) > 1 {
+		arg = os.Args[1]
+	}
+	if arg == "-h" {
+		fmt.Println("go-ls : list directory contents")
```

Comment:
We should use `flag` package to parse flags more reliably, what if the user provided more args as well as `-h` option?